src/Features/Core/Portable/MakeFieldReadonly/AbstractMakeFieldReadonlyCodeFixProvider.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Formatting;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.MakeFieldReadonly
+{
+    internal abstract class AbstractMakeFieldReadonlyCodeFixProvider
+        <TSymbolSyntax, TFieldDeclarationSyntax>
```

Comment:
please put typeparameters on previous line.  (wrapping is ok if they're really long).  that doesn't apply here.

Reply:
➡️ Fixed in commit 38db528